### ISSUE TITLE
--max-old-space-size=4096 leads to electron crash when trying to run debug build

### ISSUE THREAD

**Issue by IHappyDayI** [tagspaces/tagspaces#886]
Commit 046853621f159f3c9490fd2fbff908e8fa492049, which introduces the --max-old-space-size=4096 flag leads to the app crashing immediately on start when trying to run the app with "yarn dev" or "yarn start".
> error Command failed with exit code 3221225477

This seems to be a known issue.
https://github.com/Microsoft/vscode/issues/43223#issuecomment-364164972
https://github.com/NativeScript/nativescript-cli/issues/3652#issuecomment-397211041
https://github.com/jhipster/generator-jhipster/issues/6753#issuecomment-347446572

I have reverted that commit for my own dev environment, but I thought I'll let you know in case someone else runs into this.

A built and packaged app still runs fine, even with the flag.

**Comment by IHappyDayI** [comment]
I'm on win7 btw.

**Comment by uggrock** [comment]
Interesting on Windows 10 with 8 gb of RAM it is working fine...

**Comment by IHappyDayI** [comment]
I have 16 GB of RAM. Out of curiosity, were you running into a memory issue or why did you decide to add the flag?

**Comment by uggrock** [comment]
The idea of adding this parameter was to increase the amount files which TagSpaces can show in one single directory. Currently it has issues displaying more then 1000 files. I think soon or later we should add some kind of pagination.

**Comment by IHappyDayI** [comment]
Virtual lists sound like a solution. Only render the entries that can actually be seen on the screen. There seem to be several libraries for JS and React out there.

**Comment by uggrock** [comment]
Yes, there are some solutions like react-virtualized, which will be probably just fine for the list view, the questions is, is there anything similar for the grid view?

**Comment by uggrock** [comment]
I will close this issue since the code causing the problem is removed.
